includes/admin/settings/class-wc-settings-accounts.php

```diff
@@ -171,6 +173,26 @@ public function get_settings() {
 					'type' => 'sectionend',
 					'id'   => 'privacy_policy_options',
 				),
+				array(
+					'title' => 'Usage Tracking',
```

Comment:
@mikejolley you probably will like to review this change, since is moving the tracking into Settings -> Accounts.